front/src/applications/operationalStudies/views/v2/SimulationResultsV2.tsx

```diff
@@ -187,14 +203,32 @@ const SimulationResultsV2 = ({
               pathProperties={pathProperties}
               sharedXScaleDomain={positionScaleDomain}
               setSharedXScaleDomain={setPositionScaleDomain}
-              departureTime={selectedTrain.start_time}
+              departureTime={selectedTrainSchedule.start_time}
             />
           </div>
         </div>
       )}
 
+      {/* TIME STOPS TABLE */}
+      {selectedTrainSchedule &&
+        trainSimulation.status === 'success' &&
+        pathProperties &&
+        infraId && (
+          <div className="osrd-simulation-container mb-2">
+            <TimesStopsOutput
+              simulatedTrain={trainSimulation}
+              pathProperties={pathProperties}
+              operationalPoints={operationalPoints}
+              selectedTrainSchedule={selectedTrainSchedule}
+              startTime={startTime}
```

Comment:
StartTime should be the selectedTrainSchedule start_time, and not the one kept in the store (it's the one used to ManageTrainSchedule, to create or edit a train)
```suggestion
              startTime={selectedTrainSchedule.start_time}
```